### ISSUE TITLE
Lib namming cause TS config error during import

### ISSUE THREAD

**Issue by Asthonishia** [mkolsv/circle-progress.vue#11]
You should rename the library ( folders and files ) from circle-progress.vue to circle-progress-vue for exemple.
Actually, during import, it generates problems like this is one with the TS configuration ( shims-vue.d.ts  ) : 

"error TS2614: Module '"*.vue"' has no exported member 'CircleProgressBar'. Did you mean to use 'import CircleProgressBar from "*.vue"' instead?"

And a derived problem in IDE "Cannot resolve symbol 'CircleProgressBar''.

**Comment by mrleblanc101** [comment]
Indeed, having an extension in the package name seems like a bad idea to me
